test/test_infectionwithfeedback_plate_compatibility.py

```diff
@@ -0,0 +1,44 @@
+"""
+Test the InfectionsWithFeedback class works well within numpyro plate
+"""
+
+import jax.numpy as jnp
+import numpy as np
+import numpyro
+import numpyro.distributions as dist
+
+import pyrenew.latent as latent
+from pyrenew.deterministic import DeterministicPMF
+from pyrenew.randomvariable import DistributionalVariable
+
+
+def test_infections_with_feedback_plate_compatibility():
+    """
+    Test the InfectionsWithFeedback matching the Infections class.
+    """
+    I0 = jnp.array(
+        np.array([0.0, 0.0, 0.0, 0.5, 0.6, 0.7, 0.8] * 5).reshape(-1, 5)
+    )
+    Rt = jnp.ones((10, 5))
```

Comment:
Can we also test that a scalar `Rt` gets successfully broadcast?

Reply:
Not sure I understand! 1-dimensional `Rt` does not get successfully broadcast in the `plate` context with multidimensional `I0`.

Reply:
I just tried it out and I see what you mean. It should be doable but we'll need to do a bit more refactor (including in how we broadcast the feedback strength. But that's out of scope for this PR. So I think this can land and we can make a new issue for further generalization / vectorization and associated testing.